From cea49f4171727091b215f830819f5cda31991d2d Mon Sep 17 00:00:00 2001
From: maxsu <your_email@mail.com>
Date: Mon, 10 Jul 2017 23:24:10 +0800
Subject: [PATCH] Added my_strcat function

---
 src/string.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/string.py b/src/string.py
index 42fd1dd..2e9cbdc 100644
--- a/src/string.py
+++ b/src/string.py
@@ -4,4 +4,7 @@ var1 = 'Hello World!'
 var2 = "Python Programming"
 
 print ("var1[0]: ", var1[0])
-print ("var2[1:5]: ", var2[1:5]) # 切片加索引
\ No newline at end of file
+print ("var2[1:5]: ", var2[1:5]) # 切片加索引
+
+def my_strcat(str1, str2):
+	return (str1+str2)
-- 
1.9.5.msysgit.1

